docs(docs-infra): improve clarity of toh-pt2 li>button code (#45858)
angular/angular: aio/content/tutorial/toh-pt2.md

1.  Add an `<h2>` at the top.
1.  Below it add an HTML unordered list \(`<ul>`\) element.
1.  Insert an `<li>` within the `<ul>`.

> 1.  Place a `<button>` inside the `<ul>` that displays properties of a `hero` inside `<span>` elements.

Reviewer: ```suggestion
1.  Place a `<button>` inside the `<li>` that displays properties of a `hero` inside `<span>` elements.
```

Author: sorry copy-paste mistake :disappointed:

After the browser refreshes, the list of heroes appears.

<div class="callout is-helpful">

<header>Interactive elements</header>

**NOTE**: <br />

> Inside the `<li>` element, we've wrapper the hero's details in a `<button>` element, we did so because we will later make such hero clickable, and it is better for accessibility purposes to use natively interactive HTML elements instead of adding event listeners to non-interactive ones.

Reviewer: ```suggestion
Inside the `<li>` element, we've wrapped the hero's details in a `<button>` element. Later on we make the hero clickable, and it is better for accessibility purposes to use natively interactive HTML elements (e.g. `<button>`) instead of adding event listeners to non-interactive ones (e.g. `<li>`).
```

Author: Ah much better thanks! :slightly_smiling_face: 

(and sorry about the wrappeR typo :disappointed:)
